### PR TITLE
Update release procedure

### LINKED ISSUE #52: Consistent compression algorithm and naming for released image name
We should keep the released image names consistent and the compression algorithms consistent. Looking at the current pre-releases, it seems like the the xz algorithm is more efficient than 7z. 

The release tags should also be consistent. We should keep these constant so the documentation (#51) can refer to a consistent image.

Since we are updated to Ubuntu 22.04.1, I can create a new release soon.

### PR DESCRIPTION
**Pull request by shuhaowu**
@LanderU @carlossvg 

I created some docs so we can standardize the release procedures. I'll follow this guide once merged and create a release. I'll base my beginners doc on that release!

Also updated the latency histogram as the new image is now based on ubuntu 22.04.1. My intended release is going to be `22.04.1_v5.15.39-rt42-raspi_ros2_humble`.

Resolves #52

### CODE REVIEW

File: docs/ReleaseImage.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+   `cyclictest` duration is 2 hours for now). This will save test results into
+   the `data` folder as a `.log` file. You can detach the tmux session by
+   pressing the key sequence: `CTRL+b`, then `d`. To get back into this
+   session, run `tmux attach` once you connect back to the Raspberry Pi 4.
+6. In the same directory as above, run `./plot_latency.sh`. This will generate
+   a PNG file from the test results.
+7. Copy these files back to your local machine and commit them in the git repo.
+8. Create a PR with these files. Ping for reviews and merge the PR.
+9. Create a tag based on this, named
+   `{ubuntu_version}_v{kernel_version}_ros2_{ros_release}`. The kernel version is An example is:
```
Comment (LanderU) [inline docs/ReleaseImage.md:27]:
```suggestion
   `{ubuntu_version}_v{kernel_version}_ros2_{ros_release}`. The kernel version is an example is:
```

File: docs/ReleaseImage.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+   `22.04.1_v5.15.39-rt42-raspi_ros2_humble`. The `ubuntu_version` is based on
+   the version string of the image we download, and the `kernel_version` is the
+   output of `uname -r`.
+10. Push the tag to the repo.
+11. Take the image you built, `xz` it via the command: `cd out; sudo xz -k ubuntu-22.04.1-rt-ros2-arm64+raspi.img`.
+12. [Create a release based on the tag](https://github.com/ros-realtime/ros-realtime-rpi4-image/releases/new). Upload the gzip'ed image. Follow the following template for the release notes:
```
Comment (LanderU) [inline docs/ReleaseImage.md:33]:
```suggestion
12. [Create a release based on the tag](https://github.com/ros-realtime/ros-realtime-rpi4-image/releases/new). Upload the gzip'ed image. Follow the below template for the release notes:
```

Comment (carlossvg) [reply docs/ReleaseImage.md:33]: Are we using gzip?

Comment (shuhaowu) [reply docs/ReleaseImage.md:33]: True. `xz` takes a bit more time but makes the download faster. I'll change the wording.

**Comment by shuhaowu** [comment]
I'm just reviewing through the issues list, and I think these should be investigated before we create a final release:

- https://github.com/ros-realtime/ros-realtime-rpi4-image/issues/28
- https://github.com/ros-realtime/ros-realtime-rpi4-image/issues/38

Not sure if you guys have time for this. If not, I can maybe look at it soon (that said I'm going on vacation starting Sept 8..)

**Review by LanderU — APPROVED** [review]
LGTM!

**Comment by LanderU** [comment]
@shuhaowu 

>>>
I'm just reviewing through the issues list, and I think these should be investigated before we create a final release:

https://github.com/ros-realtime/ros-realtime-rpi4-image/issues/28

    https://github.com/ros-realtime/ros-realtime-rpi4-image/issues/38

Not sure if you guys have time for this. If not, I can maybe look at it soon (that said I'm going on vacation starting Sept 8..)
>>>

I don't think I'll have time for this, sorry. I can try but, I cannot promise.

**Comment by shuhaowu** [comment]
That's fine. Thanks! I'll try to take a look today. It might be pretty easy to get it working.

**Pull request merged**